.github/scripts/auto-label.cjs

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const path = require('path');
+
+const babelParser = require('recast/parsers/babel');
+const json5 = require('json5');
+const lodash = require('lodash');
+const recast = require('recast');
+const github = require('@actions/github');
+const { HttpClient } = require('@actions/http-client');
+
+const prefixLabelMap = {
+  'resources/': 'resources',
+  'util/': 'util',
+  'test/': 'test',
+  '.mocharc.cjs': 'test',
+  'eslint/': 'style',
+  '.eslintrc.js': 'style',
+  '.github/workflows/': 'ci',
+  '.github/scripts/': 'ci',
+
+};
+
+const getLabels = async (github, owner, repo, pullNumber) => {
+  const httpClient = new HttpClient();
+  const { data: pullRequest } = await github.rest.pulls.get({
+    owner,
+    repo,
+    // eslint-disable-next-line camelcase
+    pull_number: pullNumber,
+  });
+  if (!pullRequest.mergeable) {
+    console.log('not mergeable PR, skip');
+    return;
+  }
```

Comment:
Only `maybeTimelineReplaceChange` doesn't work if the patch is not mergeable, correct? Could we still do the other checks and apply labels even for a non-mergeable patch?

Reply:
So which committs to compare for non-mergeable PR? 

It's impossible to compare with base branch and head branch, Maybe just all PR commits?

Reply:
I think we can compare all commits in this PR if it's not mergeable.